mule-packager/src/test/java/org/mule/tools/api/ContentGeneratorTest.java

```diff
@@ -36,9 +36,9 @@
 import org.mule.tools.api.classloader.model.Artifact;
```

Comment:
It seems this class is validating MuleContentGenerator please change the name to MuleContentGeneratorTest

Also please create a DomainBoundleContentGeneratorTest